runtime/common/src/benchmarking.rs

```diff
@@ -15,13 +15,15 @@
 // along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
 
 use account::AccountId20;
+use frame_support::traits::fungible::NativeOrWithId;
+use moonbeam_core_primitives::AssetId;
 use pallet_treasury::ArgumentsFactory;
 
 pub struct BenchmarkHelper;
 
-impl ArgumentsFactory<(), AccountId20> for BenchmarkHelper {
-	fn create_asset_kind(_seed: u32) -> () {
-		()
+impl ArgumentsFactory<NativeOrWithId<AssetId>, AccountId20> for BenchmarkHelper {
+	fn create_asset_kind(_seed: u32) -> NativeOrWithId<AssetId> {
+		NativeOrWithId::Native
```

Comment:
Preferably we should benchmark for the `NativeOrWithId::WithId` since it would be the worst case scenario. (Needs to call evm, etc...)